pdp/static/js/hydro_stn_controls.js

```diff
@@ -94,7 +94,7 @@ function addToSidebar(idx, dataArray) {
 
     link = document.createElement('a');
     const data_dir = isArchivePortal() ? "hydro_stn_archive" : "hydro_stn_cmip5";
-    link.href = pdp.data_root + "/" + dataURL() + "/" + dataArray[idx].FileName + '.ascii';
+    link.href = pdp.data_root + "/" + dataURL() + "/" + dataArray[idx].FileName;
```

Comment:
Just FYI, in JS there is a thing called a [template string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) that functions very like f-strings in Python. It makes expressions like this easier to write and read.